softdevice_controller/CHANGELOG.rst

```diff
@@ -15,9 +15,12 @@ Master branch
 Added
 =====
 
+* Added support for multiple advertising sets (DRGN-15426).
 * Added radio front-end module (FEM) support, based on the :ref:`mpsl_fem` (DRGN-11059).
 * Added support for the vendor specific HCI command: Read Supported Vendor Specific Commands (DRGN-13763).
 * Added support for the vendor specific HCI command: Zephyr Read Key Hierarchy Roots (DRGN-13237).
+* On nRF53 Series devices, the maximum supported radio output power is increased from 0 dBm to 3 dBm.
```

Comment:
Should this not go under Changes?

Reply:
```suggestion
* Increased the maximum supported radio output power on nRF53 Series devices from 0 dBm to 3 dBm.
```

Reply:
- [ ] Move "Increased the maximum supported radio output power on nRF53 Series devices from 0 dBm to 3 dBm." to "Changes"